lxd/device/gpu_sriov.go

```diff
@@ -83,25 +83,53 @@ func (d *gpuSRIOV) Start() (*deviceConfig.RunConfig, error) {
 	saveData := make(map[string]string)
 
 	// Get SRIOV parent, i.e. the actual GPU.
-	parentPCIAddress, err := d.getParentPCIAddress()
+	parentPCIAddresses, err := d.getParentPCIAddresses()
 	if err != nil {
 		return nil, err
 	}
 
-	// Get PCI information about the GPU device.
-	devicePath := filepath.Join("/sys/bus/pci/devices", parentPCIAddress)
+	var parentPCIAddress string
+	var pciParentDev pcidev.Device
+	var vfID int
```

Comment:
@monstermunchkin probably worth initializing this to -1 to provide protection in case `getParentPCIAddresses` ever returns an empty slice with a nil error.